source/admin_manual/list_of_events.rst

```diff
@@ -1791,3 +1795,79 @@ happens.
 +-----------------------+------------------------------------------------------+
 | object_id             | Object ID in the storage                             |
 +-----------------------+------------------------------------------------------+
+
+Dictionaries
+============
+
+.. versionadded:: 2.3.11
+
+Events emitted by dictionary drivers and dictionary server.
+
+.. _event_dict:
+
+Common fields for dict events
+-----------------------------
+
++----------------+------------------------------------------------------+
+| Field          | Description                                          |
++================+======================================================+
+| driver         | Name of the dictionary driver                        |
```

Comment:
A couple of examples would also be useful. Especially that I think this is "client" when dict proxying is being used?